backend/Makefile

```diff
@@ -112,28 +71,6 @@ run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go --zap-log-level debug 
 	# more on controller log level configuration: https://sdk.operatorframework.io/docs/building-operators/golang/references/logging/
 
```

Comment:
I would say let's leave in the removed lines in the file, but comment them out, similar to what was done for `cluster-agent/Makefile`. It makes the Makefile a bit messier, but it makes it easier for us to revive the OLM logic if we need to.

Reply:
done ;)